src/cljs/rems/application.cljs

```diff
@@ -1003,8 +990,8 @@
   (let [app-id (:application/id application)
         ;; The :see-everything permission is used to determine whether the user
         ;; is allowed to see all comments. It would not make sense for the user
-        ;; to be able to write a comment which he then cannot see.
-        show-comment-field? (contains? (:application/permissions application) :see-everything)
+        ;; to be able to write a comment which they cannot see.
+        show-comment-field? (application-util/see-everything? application)
```

Comment:
We should consider renaming "see-everything" to something like "is an applicant or member" and "is a handling person". These are the two sides of the process and we could rewrite some conditions to be clear with these.

Reply:
I don't have good suggestions for the name but at least there could be in the likes of `applying-users` or `handling-side` for example.

Reply:
We can add these to the glossary also.

Reply:
let's add an idea about this :+1:

Reply:
implemented rudimentary `applying-users` and `handling-users` in this PR, which can be further used